tests/conftest.py

```diff
@@ -72,25 +73,20 @@ def assert_request_was_not_made(query):
             queries.append(request.querystring)
 
     @staticmethod
-    def reset():
-        httpretty.disable()
+    def run_and_collect(call, *args, **kwargs):
```

Comment:
The method name should mention assert as well
IMO, "call" should be renamed to "function"

Reply:
As an alternative, we can accept just callable and let users of this function handle arguments themselves. Like this:

```python
http_check.run_and_collect(partial(universum.run, git_poll_environment.settings))
```